### PR TITLE
chore: add chat example test

### PR DESCRIPTION
**Pull request by vasco-santos**
Add chat example test and CI job.
`libp2p-bundle` file was also renamed into `libp2p`

### CODE REVIEW

File: examples/chat/test.js
Comment (hugomrdias) [inline examples/chat/test.js:13]: i would add an `onData` as a param, do a the `.all.on('data', onData)` here and avoid the pDefer below

**Review by hugomrdias — APPROVED** [review]
LGTM, left a suggestion if you feel it makes sense.

File: examples/chat/test.js
Comment (vasco-santos) [reply examples/chat/test.js:13]: That is a good suggestion. However in this case we have to coordinate multiple parts. In the listener side, we need to wait for the listener to be ready to run the dialer process, but we also need to wait for a message to be received in the same process. This would not be possible if we block this waiting for an answer

**Pull request merged**
